index.html

```diff
@@ -29,6 +29,27 @@ <h2 class="section__title">Background Box</h2>
           // codul la tema despre background-box
         </div>
       </div>
+      <div class="section" id="background-box">
+        <h2 class="section__title">Background Box</h2>
+        <div class="section__content">
+          <div class="background-box"></div>
+          // codul la tema despre background-box
+        </div>
+      </div>
+      <div class="section" id="background-box">
+        <h2 class="section__title">Background Box</h2>
```

Comment:
New section name

Reply:
@macovei1337 what should be the new name?